python/grizzly/grizzly/grizzly.py

```diff
@@ -5,6 +5,139 @@
 from lazy_op import LazyOpResult, to_weld_type
 from weld.weldobject import *
 
+class DataFrameWeldExpr:
+    def __init__(self, expr, column_names, weld_type, is_pivot=False):
```

Comment:
I needed to introduce this class to be able to operate on dataframes that were tied to code that basically zipped their columns. I.e. some sort of zip expression.

Introducing another class just to do this is probably not the best option and we should find a way to merge the code here back into the original DataFrameWeld class.